test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterImporterEoriNumberControllerSpec.scala

```diff
@@ -170,9 +181,21 @@ class EnterImporterEoriNumberControllerSpec
 
         val updatedJourney = fillingOutClaim.copy(draftClaim = draftC285Claim)
 
+
+        val dd: DisplayDeclaration = sample[DisplayDeclaration]
```

Comment:
maybe `val displayDeclaration = sample[DisplayDeclaration]` is better?
`dd` is not great naming and no need fro explicit type defination